server/lib/db.js

```diff
@@ -10,11 +10,11 @@ module.exports = {
     }
     return new Promise(function (resolve, reject) {
       var conn = mysql.createConnection({
-        host: config.get('/db/host'),
-        port: config.get('/db/port'),
-        user: config.get('/db/user'),
-        password: config.get('/db/pass'),
-        database: config.get('/db/name'),
+        host: 'db',
```

Comment:
this is meant to be run in docker linked to the database container. if it's important people can run this app independently of docker, they can set `db` to be `localhost` in their `/etc/hosts` file and set some additional env vars or we can update this code to be more generic and accommodating